### PR TITLE
Adding my question to avoid future merge conflicts

### PR DESCRIPTION
**Pull request by wpj0110**

### CODE REVIEW

**Comment by wpj0110** [comment]
Found the error. I didn't `git add` all the files that I'm supposed to. I was going to add them later because I was going to make my own changes, but I suppose this will do for now.

File: app/components/questions/RentBurdenRates.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      <select onChange={handleChange}>
+        {props.data.metrics.map((d, i) => (
+          <option key={i} value={d.area_number}>{d.name}</option>
+        ))}
+      </select>
+    </div>
+  );
+}
+
+function AreaBelongingMessage(props) {
```
Comment (vingkan) [inline app/components/questions/RentBurdenRates.js:165]: This still says belonging, do a quick sweep through the file to see if there are any other places that need to be updated to rent burdened.

**Review by vingkan — APPROVED** [review]
I left one comment for you to look into, but otherwise looks good to merge! I am not at my laptop to pull and try the change, but the automated tests say the site still builds, so you're good to deploy!

File: app/components/questions/RentBurdenRates.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        <p>What percentage of households in {areaData.name} are rent burdened?</p>
+        <FailureNotification error={error} data={data} />
+      </div>
+      <div className="center">
+        <ExamplePieChart data={chartData} />
+        <br />
+      </div>
+      <div className="center medium-width">
+        <p>
+        <span>Based on data from 2019, {areaData.rentBurdenRateFormatted} of households in {areaData.name} are rent burdened.</span>
```
Comment (vingkan) [inline app/components/questions/RentBurdenRates.js:211]: Oh I realized just now that you're not even using the AreaRentBurdenedMessage element you have defined above. In a later PR you can replace this HTML with that component.

**Pull request merged**